src/python/espressomd/io/writer/h5md.py

```diff
@@ -42,19 +40,25 @@ class UnitSystem:
         `here <https://nongnu.org/h5md/modules/units.html#unit-string>`,
         e.g. ``UnitSystem(time='ps', mass='u', length='nm', charge='e')``.
         """
-        mass: str
-        time: str
-        length: str
-        charge: str
-        force: str = dataclasses.field(init=False, default='')
-        velocity: str = dataclasses.field(init=False, default='')
-
-        def __post_init__(self):
+
+        def __init__(self, **kwargs):
```

Comment:
you could still use type annotations if you wanted to...

Reply:
there's no need to re-implement dataclasses :) providing a different type already throws a meaningful error message: `RuntimeError: Provided argument of type int is not convertible to std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >`